src/NuGet.Services.BasicSearch.Cloud/NuGet.Services.BasicSearch.Cloud.ccproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>2.7</ProductVersion>
+    <ProductVersion>2.9</ProductVersion>
```

Comment:
We can do the upgrade (but that requires upgrading all build servers as well). I wish the SDK was a NuGet package as all we need is one DLL... Your call :-)